src/components/screens/question-form.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardActionArea,
+  CardActions,
+  CardContent,
+  Checkbox,
+  TextField,
+  Typography
+} from "@mui/material";
+import type React from "react";
+import { useState } from "react";
+
+/**
+ * NewQuestionCard component
+ * Interface for the NewQuestionCard component
+ */
+
+interface NewQuestionProps {
+  handleAddQuestionSubmit: (
+    questionText: string,
+    options: { label: string; value: boolean }[]
+  ) => void;
+}
+
+const NewQuestionCard: React.FC<NewQuestionProps> = ({ handleAddQuestionSubmit }) => {
+  /**
+   * State for the question text and options
+   */
+  const [questionText, setQuestionText] = useState("");
+  const [options, setOptions] = useState([
+    { label: "", value: false },
+    { label: "", value: false },
+    { label: "", value: false },
+    { label: "", value: false }
+  ]);
+
+  /**
+   * Handle options label change
+   */
+  const handleAnswerLabelChange = (
+    index: number,
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
```

Comment:
Destructure the react imports so you can use ChangeEvent here rather than React.ChangeEvent. This should be done in all places (unless doing so causes type problems).